values.yaml

```diff
@@ -67,7 +67,21 @@ st2:
   config: |
     [api]
     allow_origin = '*'
-
+  
+  #Override Definitions can be added here. 
+  #  overrides:
+  #    _global.yaml: |
+  #    ---
+  #    rules:
+  #      defaults:
+  #        enabled: true
+  #  packA.yaml: |
+  #    ---
+  #    rules:
+  #      rule.name:
+  #        enabled: false
```

Comment:
I'd also prefer if we'll place this commented example after the `overrides: {}` line.
That would be consistent with the other examples in Helm values.

Reply:
```suggestion
  #  overrides:
  #    _global.yaml: |
  #      ---
  #      rules:
  #        defaults:
  #          enabled: true
  #    packA.yaml: |
  #      ---
  #      rules:
  #        rule.name:
  #          enabled: false
```

Reply:
Made the changes here to place the example below `overrides`